mods/d2k/sequences/misc.yaml

```diff
@@ -314,17 +314,19 @@ crate:
 
 # TODO: keep the redundant spicebloom.shp for now for the awful WinForms maps editor
```

Comment:
We already axed it in https://github.com/OpenRA/OpenRA/pull/8511. I forgot to remove this line and `mods/d2k/bits/spicebloom.shp`. Would you be so kind and do it here?